client/src/ui/deckbuilder/DeckBuilderExportMenu.vue

```diff
@@ -56,34 +68,48 @@ export default Vue.extend({
     },
 
     exportedDeck(): string {
-      return this.deck ? deckToXml(this.deck) : '';
+      return this.deck ? exportLoader.chunk.deckToXml(this.deck) : '';
     },
 
     exportedBinder(): string {
-      return this.deck ? deckToBinderXml(this.deck) : '';
+      return this.deck ? exportLoader.chunk.deckToBinderXml(this.deck) : '';
     },
   },
 
   asyncComputed: {
     async exportedDecksZip(): Promise<string> {
-      return await decksToBinderZip(store.decks, store.names);
+      if (this.libLoaded) {
+        return await exportLoader.chunk.decksToBinderZip(
+            store.decks, store.names);
+      } else {
+        return '';
+      }
     }
   },
 
   methods: {
+    loadExportLib() {
+      exportLoader.load().then(() => {
+        this.libLoaded = true;
+      })
+    },
+
     exportToPdf() {
       if (this.deck) {
-        deckToPdf(this.deck);
+        exportLoader.chunk.deckToPdf(this.deck);
       }
-    }
+    },
   },
 });
 </script>
 
 <style scoped>
 ._deck-builder-export-menu {
-  background: white;
-  box-shadow: #a5a5a5 3px 2px 2px;
+  background-color: #FFF;
+  border-radius: 5px;
+  box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.3);
+  border: 1px solid #ccc;
+  min-width: 150px;
```

Comment:
I fixed this in my PR

Reply:
Nice! Updating the values a little to match the existing popovers.